scipy/stats/tests/test_qmc.py

```diff
@@ -320,36 +321,32 @@ def test_primes(self):
 
 class TestVDC:
     def test_van_der_corput(self):
-        seed = np.random.RandomState(12345)
-        sample = van_der_corput(10, seed=seed)
+        sample = van_der_corput(10)
```

Comment:
Pass in `rng` here, no?

Reply:
VDC by default is not scrambled so no rng is used in the background and the sequence is deterministic.

Reply:
VDC by default is not scrambled so no rng is used in the background and the sequence is deterministic.

Reply:
So `seed` was unused and shouldn't really have been here before. OK.